apps/portal/src/components/fixed/inbox/InboxOvertimeContent.tsx

```diff
@@ -4,54 +4,63 @@ import { AlertNotification, Button, Modal } from '@gscwd-apps/oneui';
 import { useInboxStore } from '../../../store/inbox.store';
 import { PsbMembers } from 'apps/portal/src/types/inbox.type';
 import { InboxMessageResponse } from 'libs/utils/src/lib/enums/inbox.enum';
+import { EmployeeOvertimeDetail } from 'libs/utils/src/lib/types/overtime.type';
+import dayjs from 'dayjs';
 
 export const InboxOvertimeContent = () => {
-  const { setConfirmModalIsOpen, setSelectedVppId, setConfirmationResponse, setDeclineRemarks } = useInboxStore(
-    (state) => ({
+  const { overtimeDetails, setConfirmModalIsOpen, setSelectedVppId, setConfirmationResponse, setDeclineRemarks } =
+    useInboxStore((state) => ({
+      overtimeDetails: state.message.overtime,
       setConfirmModalIsOpen: state.setConfirmModalIsOpen,
       setSelectedVppId: state.setSelectedVppId,
       setConfirmationResponse: state.setConfirmationResponse,
       setDeclineRemarks: state.setDeclineRemarks,
-    })
-  );
+    }));
 
   return (
     <>
-      <div className={'w-100 pl-8 pr-8 pt-1 flex flex-col bg-white pb-10'}>
+      <div className={'w-100 pl-8 pr-8 pt-1 flex flex-col pb-10'}>
+        <AlertNotification alertType="info" notifMessage={'Overtime Notification'} dismissible={false} />
         <label className="pb-2">
           This is to inform you that you have been requested for Overtime with the specified details below:
         </label>
         <div>
           <label className="font-bold">Date: </label>
-          September 30, 2023
+          {dayjs(overtimeDetails.plannedDate).format('MM-DD-YYYY')}
```

Comment:
create a function that would catch first if overtimeDetails.plannedDate is not empty to avoid out put error. If the value doesn't have content, it would show the current date today.